packages/kit/src/routes/config/getStateFromPath.ext.ts

```
@@ -139,74 +152,197 @@ export function getStateFromPath<ParamList extends {}>(
+
+  if (current == null || result == null) {
+    return undefined;
+  }
+
+  return result;
+}
+
+/**
+ * Reference to the last used config resources. This is used to avoid recomputing the config resources when the options are the same.
+ */
+const cachedConfigResources = new WeakMap<Options<{}>, ConfigResources>();
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)